mne/commands/mne_setup_source_space.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+"""Set up bilateral hemisphere surface-based source space with subsampling.
+
+Examples
+--------
+.. code-block:: console
+
+    $ mne setup_source_space --subject sample
+
+
+ .. note : Only one of --ico, --oct or --spacing options can be set at the same
+           time. Default to spacing = 7mm.
+
+"""
+
+import sys
+
+import mne
+
+
+def run():
+    """Run command."""
+    from mne.commands.utils import get_optparser, _add_verbose_flag
+    parser = get_optparser(__file__)
+
+    parser.add_option('-s', '--subject',
+                      dest='subject',
+                      help='Subject name (required)',
+                      default=None)
+    parser.add_option('--src', dest='fname',
+                      help='Output file name. Use a name <dir>/<name>-src.fif',
+                      metavar='FILE', default=None)
+    parser.add_option('--morph',
+                      dest='subject_to',
+                      help='morph the source space to this subject',
+                      default=None)
+    parser.add_option('--surf',
+                      dest='surface',
+                      help='The surface to use. (default to white)',
+                      default='white',
+                      type='string')
+    parser.add_option('--spacing',
+                      dest='spacing',
+                      help='Specifies the approximate grid spacing of the '
+                           'source space in mm. (default to 7mm)',
+                      default=None,
+                      type='int')
+    parser.add_option('--ico',
+                      dest='ico',
+                      help='use the recursively subdivided icosahedron '
+                           'to create the source space.',
+                      default=None,
+                      type='int')
+    parser.add_option('--oct',
+                      dest='oct',
+                      help='use the recursively subdivided octahedron '
+                           'to create the source space.',
+                      default=None,
+                      type='int')
+    parser.add_option('-d', '--subjects-dir',
+                      dest='subjects_dir',
+                      help='Subjects directory',
+                      default=None)
+    parser.add_option('--cps',
+                      dest='cps',
+                      help='Add patch information to source space.',
+                      default=True,
+                      action="store_true")
```

Comment:
so we don't remove this option then?

Reply:
Removed, but I didin't specifically set add_dist=True, so the command line will always use the default python option.